src/Raven.Studio/typescript/common/queryCompleter.ts

```diff
@@ -308,7 +309,9 @@ class queryCompleter {
         this.getIndexFields(queryIndexName.name, queryIndexName.type, prefix)
             .done((wordList) => {
                 if (functions) {
-                    wordList = wordList.concat(functions);
+                    functions.forEach(f => {
+                        wordList.push(f);
```

Comment:
You can use:
wordList.push(...functions)

Reply:
Thanks.